transform.js

```diff
@@ -73,7 +74,8 @@ function transform (filename, options) {
     var mname = null
     var ast
 
-    if (src.indexOf('sheetify') === -1) {
+    // Skip transforming any files that do not contain synonyms for sheetify
+    if (!SUPPORTED_VIEWS.some(name => src.indexOf(name) > -1)) {
```

Comment:
You can also use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes to make intent just a little bit clearer

Reply:
Ah yeah, agreed.  Let me fix that.